site/tutorials/streaming.md

```diff
@@ -91,7 +91,7 @@ vegaEmbed('#chart', vlSpec).then(function (res) {
 });
 ```
 
-As a word of caution, in some cases, streaming data might cause some unexpected results when rendering. If your rendered chart overflows your container in the presence of new incoming data, for example, you may need to configure [Autosize](https://vega.github.io/vega-lite/docs/size.html#autosize) functionality or explicitly use the [view.resize](https://vega.github.io/vega/docs/api/view/#view_resize) API to explicitly resize when new data is incorporated.
+As a word of caution, in some cases, streaming data might cause unexpected results when rendering. If your rendered chart overflows your container in the presence of new incoming data, for example, you may need to configure [Autosize](https://vega.github.io/vega-lite/docs/size.html#autosize) functionality or explicitly use the [view.resize](https://vega.github.io/vega/docs/api/view/#view_resize) API to explicitly resize when new data is incorporated.
```

Comment:
```suggestion
New data may change the layout but Vega does not always resize the chart. Because of this optimization, your axes may be clipped off for example. To resolve this issue, you need to configure [autosize](https://vega.github.io/vega-lite/docs/size.html#autosize) or explicitly use [view.resize](https://vega.github.io/vega/docs/api/view/#view_resize) to resize when the data updates.
```